### ISSUE TITLE
add note on notation

### ISSUE THREAD

**Issue by tabpeter** [pbreheny/plmm#19]
add an article to the website defining the mathematical notation and the naming conventions used in the package source code

**Comment by pbreheny** [comment]
Seems like this should be closed, right?

**Comment by tabpeter** [comment]
Yes, the article on notation is now complete/up on the website: https://pbreheny.github.io/plmm/articles/notation.html

**Comment by pbreheny** [comment]
This is weird, but note that when you mark something "Done" in a project, it doesn't close the issue. This issue is still open. However, when you close the issue, it will get automatically marked as "Done". It's weird, seems like a bug, but that's how it is.

In other words, please close this issue :-)

**Comment by tabpeter** [comment]
[like]  Peter, Tabitha K reacted to your message:
________________________________
From: Patrick Breheny ***@***.***>
Sent: Monday, March 4, 2024 4:41:12 PM
To: pbreheny/plmm ***@***.***>
Cc: Peter, Tabitha K ***@***.***>; Author ***@***.***>
Subject: [External] Re: [pbreheny/plmm] add note on notation (Issue #19)


This is weird, but note that when you mark something "Done" in a project, it doesn't close the issue. This issue is still open. However, when you close the issue, it will get automatically marked as "Done". It's weird, seems like a bug, but that's how it is.

In other words, please close this issue :-)

**Comment by tabpeter** [comment]
Yes, this is done
